mslice/plotting/plot_window/plot_figure.py

```diff
@@ -61,6 +62,33 @@ def _apply_config(self, plot_config):
         current_axis.set_xlim(*plot_config.x_range)
         current_axis.set_ylim(*plot_config.y_range)
 
+        images = current_axis.get_images()
+        if len(images) != 1:
```

Comment:
I think we need a similar `if not current_axis.get_images()` check here.

At the moment If I plot a cut, click plot options, uncheck show error bars and then click OK I get an error about applying the settings